app/instance-initializers/ember-a11y.js

```diff
@@ -16,12 +16,14 @@ Ember.Router.reopen({
   }
 });
 
-export function initialize(applicationInstance) {
-  if (arguments.length > 1) {
-    applicationInstance = arguments[1];
-  }
-  const container = applicationInstance.container ? applicationInstance.container : applicationInstance.__container__;
-  const application = container.lookup('application:main');
+// Method signatures:
+// 1.13: container, applicationInstance
+// 2.X: applicationInstance
+export function initialize(container, applicationInstance) {
+  lookup = container.lookup || applicationInstance.container.lookup;
```

Comment:
Does this need `let` or `const`? Looking at the test fails it looks like this is a global